### PR TITLE
Use PlantUML for Delete Client/Property Sequence Diagrams

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**

### CODE REVIEW

File: docs/diagrams/PlantUML File/DeletePropertySequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+participant "ui:Ui" as Ui UI_COLOR
+participant "storage:Storage" as Storage STORAGE_COLOR
+participant "pairingList:PairingList" as PairingList PAIRING_LIST_COLOR
+
+->CommandDeleteProperty : execute(Ui, Storage, PropertyList, ClientList, PairingList)
+activate CommandDeleteProperty COMMAND_COLOR
+
+CommandDeleteProperty -> PropertyList : deleteProperty(propertyIndex:int)
+activate PropertyList PROPERTY_LIST_COLOR
+return deletedProperty
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/DeletePropertySequenceDiagram.puml:17]: I think the return value should come in the format variable name : variable type

**Review by ngdeqi — COMMENTED** [review]
One small inaccuracy in the PlantUML diagram. Otherwise, this PR looks good.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -262,12 +262,12 @@ and their relationships.
 ![Delete Client/Property Class Diagram](diagrams/DeleteClientPropertyCD.png)
 
 The following *sequence diagram* shows how the **delete client** operation works, showcasing the
-```ClientList#deleteClient()``` method.
+```CommandDeleteClient#execute()``` method.
 
 ![Delete Client Sequence Diagram](diagrams/DeleteClientSD.png)
```
Comment (OVReader) [inline docs/DeveloperGuide.md:267]: Good job in correcting previous SD errors.

File: docs/diagrams/PlantUML File/DeletePropertySequenceDiagram.puml (hunk 1 above)
Comment (OVReader) [reply docs/diagrams/PlantUML File/DeletePropertySequenceDiagram.puml:17]: Corrected. Good job!

**Review by OVReader — APPROVED** [review]
New SDs nicely done! LGTM

**Pull request merged**